cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -345,26 +345,54 @@ def is_disposable_capital_eligible(self):
         return self.disposable_capital_assets <= limit
 
     def is_eligible(self):
-        self._do_cfe_civil_check()
-
-        legacy_result = self._legacy_check()
-        logger.info("Eligibility result (legacy): %s %s" % (legacy_result, self.calcs))
+        cfe_result, cfe_details = self._do_cfe_civil_check()
+
+        # temp: start doing just non-partner passported checks through CFE
+        if self.case_data.facts.on_passported_benefits and hasattr(self.case_data.facts,
+                                                                   "has_partner") and not self.case_data.facts.has_partner:
```

Comment:
This is how 'black' lays it out, and I think it's an improvement:
```
        if (
            self.case_data.facts.on_passported_benefits
            and hasattr(self.case_data.facts, "has_partner")
            and not self.case_data.facts.has_partner
        ):
```

Reply:
I have added some methods so that the layout problems disappear